docs/rules/eol-last.md

```diff
@@ -42,3 +42,4 @@ This rule has a string option:
 
 * `"unix"` (default) enforces line feed (LF) as newline
 * `"windows"` enforces carriage return line feed (CRLF) as newline
+* `"none"` enforces that the last line ends with neither LF nor CRLF as newline
```

Comment:
I think this might read better for non-natives if it says "enforces that the last line does not end with a LF or CRLF as newline". In fact, I'm thinking maybe we could get rid of the "LF or CLRF" part altogether and just say: "enforces that the last line does not end with a newline". What do you think?